README.md

```diff
@@ -1,55 +1,60 @@
-# FlowCrypt: Encrypt Gmail with PGP
+# FlowCrypt: Encrypt Gmail with OpenPGP
 
 ## Users
 
-Get [FlowCrypt](https://flowcrypt.com/) browser extension at: https://flowcrypt.com/download
+Get the [FlowCrypt](https://flowcrypt.com/) browser extension from the [FlowCrypt downloads](https://flowcrypt.com/download) page.
 
 ## Developers
 
 Mock tests: ![Mock tests](https://flowcrypt.semaphoreci.com/badges/flowcrypt-browser.svg?key=d575b4ff-e35d-4217-9152-27cea9e72c19)
 
-Live tests: [![Live Tests](https://semaphoreci.com/api/v1/flowcrypt/flowcrypt-browser/branches/master/badge.svg)](https://semaphoreci.com/flowcrypt/flowcrypt-browser)
+We're developing the project in TypeScript. However, since browsers only understand JavaScript, the project needs to be transpiled to it. You need to build the project the first time you download/clone it, and then build it again after each change you make to see the result in the browser. To get started, please follow the instructions below:
 
-This project is written in TypeScript. Browsers only understand JavaScript, so the project needs to be transpiled. You need to build the project the first time you download it, and build it after each change you make to see the result in the browser. To get started, please follow the instructions below:
+1. Navigate to the project folder and install the tooling by running the following commands:
 
-1. Install the tooling by running the following command after navigating to the appropriate folder:
 ```bash
+# Navigate to the appropriate folder
 $ cd some/folder/flowcrypt-browser
+# Install the tooling
 $ npm install
 ```
+
 2. To build the project (currently available for Linux and Mac only), run the following command:
 
+
 ```bash
 $ npm run build
 ```
-After running this command, you can find your built project in the `build/chrome-consumer` and `build/firefox-consumer` folders.
+
+After executing the build command, you can find your built project in the `build/chrome-consumer` and `build/firefox-consumer` folders.
+
+> Note: The `build` folder also contains other versions of the browser extension.
 
 To load the extension in Google Chrome, please follow these steps:
 
 1. Open your Chrome browser and navigate to `chrome://extensions/`.
-2. If not already enabled, toggle on the `Developer mode`    switch located at the top-right corner.
-3. Click on the `Load Unpacked` button.
-4. Browse to and select the appropriate folder, either `build/chrome-consumer` or `chrome-enterprise`.
+2. If it isn't already enabled, toggle on the **Developer mode**. The switch button is located at the top-right corner.
+3. Click on the `Load unpacked` button.
+4. Browse to the `flowcrypt-browser/build` folder and select the appropriate project version, either `build/chrome-consumer` or `chrome-enterprise`.
```

Comment:
`build/chrome-consumer` -> `chrome-consumer`
as user is already in `build` folder

Reply:
\+